setup.py

```diff
@@ -21,8 +49,4 @@
         "ipython",
         "keras==2.3.0",
     ],
-    entry_points="""
```

Comment:
@Philoso-Fish would be nice to also ofer some CLI functionality in the future, but i can see that for the moment the run.py isn't really good to release. I'll create a ticket

Reply:
Ah ok, I'm not sure if I understood the purpose of run.py correctly. At the moment it is not doing anythin important, I think. What would be a good usage for run.py?

Reply:
With the entry_points you specify the path of the package what should be run if you execute the package e.g. via 

`python -m carla` or just `carla` if you installed it on on your machine.

the run.py was merely a quick entry point when we wanted to run it. Wasn't meant for the library release, so good to remove it. In the future we can build a CLI though which arguments that could specify e.g. the benchmarking and logging behaviour.

Reply:
Ok, that makes sense, thanks.